src/ng/filter/orderBy.js

```diff
@@ -115,7 +115,7 @@
  *
  * @param {boolean=} reverse - If `true`, reverse the sorting order.
  * @param {(Function)=} comparator - The comparator function used to determine the relative order of
- *    value pairs. If omitted, the built-in comparator will be used.
+ *    value pairs. If omitted or finds two entries to be equal, the built-in comparator will be used.
```

Comment:
I think thisis more confusing than helpful. If they are interested thay read the details below.